packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-approval/index.hbs

```diff
@@ -1,19 +1,61 @@
 <ActionCardContainer
-  @header="Withdrawal"
+  @header="Confirmation"
   @isComplete={{@isComplete}}
   data-test-withdrawal-transaction-approval-is-complete={{@isComplete}}
 >
-  <ActionCardContainer::Section>
+  <ActionCardContainer::Section @title="Confirm the withdrawal details">
+    <CardPay::LabeledValue @vertical={{true}}  @label="Funding From:">
+      <CardPay::BalanceViewBanner
+        @walletAddress={{truncate-middle this.layer2Network.walletInfo.firstAddress}}
+        @depotAddress={{truncate-middle this.depotAddress}}
+        @token={{this.withdrawalToken}}
+        @balance={{format-token-amount this.tokenBalance}}
+        data-test-withdrawal-tx-approval-balance
+      />
+    </CardPay::LabeledValue>
+    <CardPay::LabeledValue @vertical={{true}} @label="Amount to Withdraw:">
+      <CardPay::BalanceDisplay
+        class="withdrawal-transaction-approval__amount-display"
+        @size="large"
+        @icon={{this.withdrawalToken.icon}}
+        @symbol={{this.tokenSymbol}}
+        @balance={{format-token-amount this.withdrawalAmount}}
+        @text={{concat "$" (token-to-usd this.convertibleSymbol this.withdrawalAmount) " USD*"}}
```

Comment:
I didn’t realise this was an option vs `@usdBalance`, I’ll adopt this in #1799 vs the unsightly `@usdFootnote` option I added, thanks.

Reply:
> I didn’t realise this was an option vs `@usdBalance`, I’ll adopt this in #1799 vs the unsightly `@usdFootnote` option I added, thanks.

I'm fine with that approach, too, if we want to follow that approach. Another idea is to extract a new component for the amount display with the border around it, since it's used in a bunch of places.